### PR TITLE
Upgrade to React 0.14

### PR DESCRIPTION
**Pull request by moretti**

### CODE REVIEW

**Comment by ovidiuch** [comment]
@moretti can you please resolve conflicts with master?

File: src/router.js [hunk 1]
```diff
@@ -75,7 +76,7 @@ Router.prototype = {
         componentElement = React.createElement(ComponentClass, props);
 
     // The router exposes the instance of the currently rendered component
-    this.rootComponent = React.render(componentElement,
+    this.rootComponent = ReactDOM.render(componentElement,
                                       this._options.container);
```
Comment (ovidiuch) [inline src/router.js]: Please align `this._options.container` with `componentElement`.

**Comment by moretti** [comment]
I've rebased my changes on top of master. Let me know if there's something else to change.
Thanks!

**Comment by ovidiuch** [comment]
Changes available under `react-querystring-router@0.3.0` — Thanks!

**Pull request merged**